Cabal/src/Distribution/Simple/ConfigureScript.hs

```diff
@@ -64,8 +77,8 @@ runConfigureScript verbosity flags lbi = do
   -- to ccFlags
   -- We don't try and tell configure which ld to use, as we don't have
   -- a way to pass its flags too
-  configureFile <-
-    makeAbsolute $ packageRoot commonFlags </> "configure"
+
+  let configureFile' = toUnix configureFile
```

Comment:
Question: why do we convert this path to unix? Is it related to the comment that just follows?

Reply:
This logic is unchanged from before, but yes I suppose you are correct.